lib/diagnostic.rb

```diff
@@ -13,27 +14,31 @@ def longest(list)
   # list supplied (i.e. apply the `+` operator).
   # Assume each entry implements a `+` method.
   def sum(list)
+    list.reduce (:+)
     # Your code goes here
   end
 
   # Question 3
   # Write ruby code that returns true if some of the entries
   # in the list of numbers are odd
   def some_odd(numbers)
+    numbers.any?{ |number| number.odd? }
     # Your code goes here
   end
 
   # Question 4
   # Write ruby code that returns true if every entry in the
   # list of numbers are even
   def every_even(numbers)
+    numbers.all? { |number| number.even?}
     # Your code goes here
   end
 
   # Question 5
   # Write ruby code that returns an array with all of the
   # entries from the list of words capitalized (i.e. 'foo' becomes 'Foo').
   def transform(words)
+    words.each { |word| word.capitalize }
```

Comment:
Again here. Short hand, which is what it's asking:

`words.map(&:capitalize)` You would use `.map` because you're changing the array itself.